Docs/INISystem.md

```diff
@@ -34,7 +34,7 @@ _NOTE: Constants can only be used in [dynamic control properties](#dynamic-contr
 - The `comma-separated strings` is a string, but separated with `,` character without spaces e.g., `one,two,three`.
 <!-- - The `comma separated integers` or `comma separated floats` is a `integer` or `float` type, but separated with `,` character without spaces e.g., `0,0` or `0.0,0.0` respectively. -->
 
-## Control properties
+## Control Properties
```

Comment:
You have modified the title as `## Control Properties` while adding a new title `## Window properties`.

Make the capital letter consistent

Reply:
Nice catch, thank you!